rfcs/datepicker-component.md

```diff
@@ -0,0 +1,130 @@
+# DatePicker Component
+
+Represents control to work and pick date, time value or a range of values
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {DatePicker} from 'baseui/datepicker';
+
+export default () => <DatePicker />;
+```
+
+### Advanced usage
+
+```javascript
+import * as React from 'react';
+import {DatePicker} from 'baseui/datepicker';
+
+export default () => {
+  return <DatePicker
+    initialState={{
+      value: [new Date(2018, 1, 15), new Date(2018, 8, 25)]
+    }}
+    isRange={true}
+    dateFormat={'MM/DD/YY'}
+    minDate={new Date(2000, 1, 1)}
```

Comment:
can you add it to the DatePicker API? also, should we add defaults too?

Reply:
thank you. That is old example, will change it soon

Reply:
@blackswanny hey!  just following up on the focus group - is this RFC ready for feedback?  I'd also be happy to contribute